src/Resources/services/factories.xml

```diff
@@ -8,6 +8,10 @@ We are hiring developers from all over the world. Join us and start your new, ex
 
 <container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
            xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+    <imports>
```

Comment:
If you're creating separate component, I would create one `services.xml` file inside, importing all dependent xmls, and then import this whole `services.xml` file in global plugin `services.xml` file.